client_dashboard/common/src/commonMain/kotlin/org/thechance/common/presentation/restaurant/RestaurantScreenModel.kt

```diff
@@ -325,7 +332,17 @@ class RestaurantScreenModel(
     }
 
     private fun onDeleteCuisinesSuccessfully(cuisineName: String) {
-        getCuisines()
+        updateState {
+            it.copy(
+                restaurantAddCuisineDialogUiState = it.restaurantAddCuisineDialogUiState.copy(
+                    isVisible = true,
+                    cuisineName = "",
```

Comment:
also i'm seeing there is a duplication code between this function and the onCreateCuisinesSuccessfully function so maybe we can make a function called resetCuisineDialogState and put the duplicated code in this function